tests/cypress/tests/e2e/profile.spec.js

```diff
@@ -0,0 +1,19 @@
+// element selectors
+const menuButton = '[data-testid="menu-button"]';
+const menuOptionManageAccount = '[data-testid="menu-option-manage-account"]';
+const bannerEditButton = '[data-testid="banner-editor-button"]';
+
+beforeEach(() => {
+  cy.visit("/");
+  cy.authenticate("adminUser");
+});
+
+describe("Profile integration tests", () => {
+  it("Navigate to profile page and then admin page", () => {
+    cy.get(menuButton).click();
+    cy.get(menuOptionManageAccount).click();
+    cy.location("pathname").should("match", /profile/);
+    cy.get(bannerEditButton).click();
+    cy.location("pathname").should("match", /admin/);
+  });
```

Comment:
I'd really like to find a better way to auto-travel to routes for integration tests in the future. It's something that's been bugging me in my own e2e tests and I've fought long and hard with `cypress-react-router`, as well as some attempts at getting it to work with custom code and just haven't found a satisfying solution yet. Something to think on. 🤔